### ISSUE TITLE
AT-14.0 failed to build on RH8

### ISSUE THREAD

**Issue by superdavidxp** [advancetoolchain/advance-toolchain#2166]
ON RHEL-8.3, we ran `make all AT_CONFIGSET=14.0` the code failed to build with error code
```
2021-06-03_17.23.27 Starting the build of glibc_1-64...
2021-06-03_17.52.09 Completed the build of glibc_1-64!
2021-06-03_17.52.09 First dynamic loader cache update...
2021-06-03_17.52.09 Completed first dynamic loader cache update!
2021-06-03_17.52.09 Starting the build of gcc_2...
make: *** [Makefile:1343: /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/gcc_2.b.rcpt] Error 1
rm /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/patch_kernel.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/patch_mpfr.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_kernel_source.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_mpc_source.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_gmp_patches.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/patch_gcc.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/patch_binutils.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_glibc_patches.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_gmp_source.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/patch_mpc.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_mpfr_patches.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_gcc_patches.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_binutils_patches.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_kernel_patches.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_mpc_patches.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/patch_gmp.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_gcc_source.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/patch_glibc.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_binutils_source.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_mpfr_source.rcpt /opt/install/tarball/advance-toolchain/at14.0-3.redhat-8_ppc64le_ppc64le/receipts/fetch_glibc_source.rcpt
```

**Comment by mscastanho** [comment]
@superdavidxp I believe this is related to #1298

**Comment by ThinkOpenly** [comment]
The fix for #1298 is now merged.
Unfortunately, it may be that builds on RHEL 8 will now hit #1969.
